package.json

```diff
@@ -125,10 +125,19 @@
     "yargs": "6.6.0"
   },
   "dependencies": {
+    "@angular/common": "4.0.0-rc.2",
```

Comment:
The fact that we don't ship anything bundled makes this kinda wonky though... :thinking: